proxy/hdrs/URL.cc

```diff
@@ -1194,6 +1194,12 @@ url_is_strictly_compliant(const char *start, const char *end)
 } // namespace UrlImpl
 using namespace UrlImpl;
 
+bool
+url_is_compliant(const char *start, const char *end)
+{
+  return url_is_strictly_compliant(start, end);
+}
+
```

Comment:
Have to do this wrapper because url_is_strictly_compliant is in the UrlImpl space.   I tried taking it out and it wouldn't build because of a missing hash table maybe because it is calling ParseRules?